read-the-docs/2/conf.py

```diff
@@ -92,6 +92,7 @@ def parse_project_name_from_pom():
     'java-api':  ('https://docs.oracle.com/en/java/javase/11/docs/api%s', None),
     'portal-url':  ('https://developer.axonivy.com/portal/' + version + '/doc%s', None),
     'github-build-examples': ('https://github.com/axonivy/project-build-examples/blob/' + buildExampleVersion + '/compile-test%s', None),
+    'api-browser-url': ('https://developer.axonivy.com/doc-api-browser?url=https://developer.axonivy.com/doc/' + version + '/public-api/%s', None),
```

Comment:
I would prefer https://developer.axonivy.com/api-browser over https://developer.axonivy.com/doc-api-browser but I need to have a look to the changes in the website.